apps/web/src/styles/theme.css

```
@@ -255,3 +255,213 @@
+@keyframes pulse {
+  0%,
+  100% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.5;
+  }
+}
+
+/* Fade in */
+@keyframes fadeIn {
```

Stylelint (17.3.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)